qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -167,7 +168,8 @@ def verify_schedule(self, dir_path, schedules, retentions=[]):
             self.assertTrue(retention in json_res['retention'])
 
     def remove_snapshots(self, dir_path):
-        snap_path = f'{dir_path}/.snap'
+        sdn = self.fs.get_config('client_snapdir')
```

Comment:
You need to set the `client_snapdir` option for ceph-fuse clients and the `snapdirname` mount option for kclient in the `.yaml` file.

For teuthology test you should use `sdn = self.mount_a.client_config.get('client_snapdir/snapdirname', ".snap")` to get the snapdir value you set in the `.yaml` file.

For vstart_runner.py local test you can continue do `sdn = self.fs.get_config('client_snapdir')`

Reply:
And also please add one dedicated `.yaml` file for your test. All the existing tests could still use the default `.snap`. I am not sure what bad things will happen if you set the custome snapdir for the existing tests.

Reply:
Made changes accordingly!!